src/pages/home/report/ReportActionsList.tsx

```diff
@@ -548,7 +602,7 @@ function ReportActionsList({
                     ref={reportScrollManager.ref}
                     testID="report-actions-list"
                     style={styles.overscrollBehaviorContain}
-                    data={sortedReportActions}
+                    data={sortedVisibleReportActions}
```

Comment:
Same data(`sortedVisibleReportActions`) we need to pass to the renderItem prop `displayAsGroup` for consistency. This inconsistency causes the issue https://github.com/Expensify/App/issues/38830 on deleting the comment.